cocos/2d/components/rich-text.ts

```diff
@@ -162,7 +162,6 @@ export class RichText extends Component {
      * 富文本显示的文本内容。
      */
     @multiline
-    @tooltip('i18n:richtext.string')
```

Comment:
Why remove it?

Reply:
use automatic addition of tooltip

code: ./app/builtin/scene/source/script/export/dump/encode.ts

<img width="1117" alt="image" src="https://github.com/user-attachments/assets/e766eecb-fe37-4c20-9815-9294dd5b3f06">